cmake/mcuboot.cmake

```diff
@@ -109,10 +109,13 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   endif ()
 
   if (DEFINED mcuboot_CONF_FILE)
-    get_filename_component(mcuboot_CONF_DIR ${mcuboot_CONF_FILE} DIRECTORY)
-    if (EXISTS ${mcuboot_CONF_DIR}/${CONFIG_BOOT_SIGNATURE_KEY_FILE})
-      set(mcuboot_key_file ${mcuboot_CONF_DIR}/${CONFIG_BOOT_SIGNATURE_KEY_FILE})
-    endif()
+    foreach(F IN LISTS mcuboot_CONF_FILE)
+      get_filename_component(mcuboot_CONF_DIR ${F} DIRECTORY)
+      if (EXISTS ${mcuboot_CONF_DIR}/${CONFIG_BOOT_SIGNATURE_KEY_FILE})
+        set(mcuboot_key_file ${mcuboot_CONF_DIR}/${CONFIG_BOOT_SIGNATURE_KEY_FILE})
```

Comment:
I would like to question if this code (both original and modified) actually works as expected.

What if `mcuboot_CONF_FILE` is defined and then user launches: `ninja mcuboot_menuconfig` and then reconfigures `BOOT_SIGNATURE_KEY_FILE` to a different file for testing ?

That doesn't seem to change the value of `mcuboot_key_file` and thing will most likely break.